mesonbuild/interpreter.py

```diff
@@ -2078,6 +2078,7 @@ def __init__(self, build, backend=None, subproject='', subdir='', subproject_dir
         self.coredata = self.environment.get_coredata()
         self.backend = backend
         self.subproject = subproject
+        self.summary = {}
```

Comment:
this should really be `collections.defaultdict(list)`

Reply:
Oh, didn't know that type, thanks!